db/migrate/20200428205535_uniqueness_rule_references_rule_index.rb

```diff
@@ -0,0 +1,28 @@
+class UniquenessRuleReferencesRuleIndex < ActiveRecord::Migration[5.2]
+  class NewRuleReferencesRule < ApplicationRecord; end
+
+  def change
```

Comment:
We should add other methods here besides `change` for readability. Also, is this _really_ reversible, or should we use `up`/`down`?

Reply:
No, the duplicate resolver should be idempotent, so running it a second time should be a no-op. By putting this in a `change` method instead of just `up`, you're telling Rails that this is reversible, but it's not - thus my comment.